doc/language_guide.md

```diff
@@ -57,15 +59,31 @@ SQLFlow will then parse the above statement and translate it to an equivalent Py
 
 ![](figures/user_overview.png)
 
-## Syntax
+## Training Syntax
+
+A SQLFlow training statement consists of a sequence of select, train, column, label, and into clauses.
 
-A SQLFlow training statement consists of a sequence of select, train, column, label and into clauses.
+```
+SELECT select_expr [, select_expr ...]
```

Comment:
select_expr -> field_expr?

Reply:
I prefer `select_expr` because it is used by MySQL syntax https://dev.mysql.com/doc/refman/8.0/en/select.html.

Reply:
I see.